### DISCUSSION TITLE
Suggestions for saving/loading the game

### DISCUSSION THREAD

**Discussion by a-senra** [dialogic-godot/dialogic#2104] [Q&A] [answered]
Hello, I'm having trouble understanding the methods for saving and loading, and I already read the documentation. I'm very inexperienced, so there are things that I don't understand well.

Right now, I want to have a button that saves the current timeline data (which timeline is the player at, which line, changes in variables, etc) to a .txt, and a button in the main menu that always loads that .txt. Right now I don't won't to go into slots and screenshots: a simple button that saves in the same .txt would do. 

I guess what I'm having more trouble is how to create that dictionary or readable .txt. I apologise if this is a more basic GDscript question, but I would like to read examples of saving fuctions similar to what I want to achieve. Thank you for your time :)

**Comment by CakeVR** [comment, marked as answer]
Hello!
There is a new page coming soon, I'm just trying to get my current PR done and merged.

All you really need to do is use `Dialogic.Save.save` and `Dialogic.Save.load`, you can ignore the slot name and leave it empty.
This will use a default slot name instead.

```gdscript
func _on_button_pressed() -> void:
    Dialogic.Save.save()
```

**Reply by a-senra** [reply]
Ok thank you, really helpful!

**Reply by CakeVR** [reply]
I'm glad to hear it was helpful information!

Let me know if you encounter issues, feel free to join the Discord too.
